pypesto/optimize/optimizer.py

```diff
@@ -623,3 +629,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class ScipyDifferentialEvolutionOptimizer(Optimizer):
+    """
+    Global optimization using scipy's differential evolution optimizer.
+    Package homepage: https://scipy.org
```

Comment:
Please add the link to the diff evo subpage. Can you also add a link for the other Scipy optimizer, to the scipy.optimize.minimize page? (https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html#scipy.optimize.minimize)

Reply:
If you mean the optimizer **class ScipyOptimizer(Optimizer)** for the second link, then yes.
Otherwise, I am not sure which other Scipy optimizer you mean.

Reply:
yes

Reply:
Done